video/agon_screen.h

```diff
@@ -59,53 +56,51 @@ inline uint8_t getVGAColourDepth() {
 void setPaletteItem(uint8_t l, RGB888 c) {
 	auto depth = getVGAColourDepth();
 	if (l < depth) {
+		// Use instance, as call not present on VGABaseController
 		switch (depth) {
-			case 2: _VGAController2.setPaletteItem(l, c); break;
-			case 4: _VGAController4.setPaletteItem(l, c); break;
-			case 8: _VGAController8.setPaletteItem(l, c); break;
-			case 16: _VGAController16.setPaletteItem(l, c); break;
+			case 2: fabgl::VGA2Controller::instance()->setPaletteItem(l, c); break;
+			case 4: fabgl::VGA4Controller::instance()->setPaletteItem(l, c); break;
+			case 8: fabgl::VGA8Controller::instance()->setPaletteItem(l, c); break;
+			case 16: fabgl::VGA16Controller::instance()->setPaletteItem(l, c); break;
 		}
 	}
 }
 
 // Change video resolution
 // Parameters:
 // - colours: Number of colours per pixel (2, 4, 8, 16 or 64)
-// - modeLine: A modeline string (see the FagGL documentation for more details)
+// - modeLine: A modeline string (see the FabGL documentation for more details)
 // Returns:
 // - 0: Successful
 // - 1: Invalid # of colours
 // - 2: Not enough memory for mode
 //
-int8_t change_resolution(uint8_t colours, char * modeLine) {
+int8_t change_resolution(uint8_t colours, char * modeLine, bool doubleBuffered = false) {
```

Comment:
changed `doubleBuffered` to be an argument, since we essentially don't really need to keep our own copy

if we do need to find out if we're double buffered, there's a call for that, which is exposed thru a function below.  as that's a reflection of the true state of the VGA controller we're better off using that.